docker/readme.md

```diff
@@ -38,3 +38,4 @@
     DICECLOUD_PASS=c
     DICECLOUD_TOKEN=d
     GOOGLE_SERVICE_ACCOUNT=e
+    SENTRY_DSN=f
```

Comment:
@madcowfred Can you open a PR to avrae-terraform to add the secret as an environment variable?

Reply:
https://github.com/avrae/avrae-terraform/pull/26